constants/admin-roles.const.js

```diff
@@ -1,3 +0,0 @@
-const adminRoles = ['core', 'maintainer', 'admin', 'moderator'];
-
-module.exports = adminRoles;
```

Comment:
I'm on the fence about the changes proposed in the PR as a whole. Is there a good reason to remove this const file?

Reply:
At least to me personally, its annoying and roundabout that there are essentially two config files that shadow each other in scope. There is no point in the code where the const file is used anymore because the code all uses the config value, but the config value is needlessly declared in a extra file and then imported into the main config file. This just adds unnecessary complexity

Reply:
I see the rationale. Let's go ahead with this change then.